wallet/src/actors/worker/methods.rs

```diff
@@ -149,21 +149,16 @@ impl Worker {
         &self,
         wallet: &types::Wallet,
         name: Option<String>,
-        caption: Option<String>,
+        description: Option<String>,
     ) -> Result<()> {
-        wallet.update(name, caption)?;
+        wallet.update(name, description)?;
 
         Ok(())
     }
 
     /// Update the wallet information in the infos database.
-    pub fn update_wallet_info(
-        &self,
-        wallet_id: &str,
-        name: Option<String>,
-        caption: Option<String>,
-    ) -> Result<()> {
-        self.wallets.update_info(wallet_id, name, caption)?;
+    pub fn update_wallet_info(&self, wallet_id: &str, name: Option<String>) -> Result<()> {
```

Comment:
I would make `update_info` require a `session_id` so that it could only be changed by an authenticated client.
Additionally, I would allow the client to change the description too (as optional param).

Reply:
As we discussed, I was wrong! :smile: